recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<p>An unification has to be made in the nb about the comment lines: either it is <code>#Blablabla</code> everywhere or <code># Blablabla</code> (my personal preference: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#comments)</p><p> </p><p>Also, start with a capital letter for ll the comment lines.</p><p> </p><p>And finally, gammapy -&gt; Gammapy in all the nb.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>